### ISSUE TITLE
How to get latest as .DMG ?

### ISSUE THREAD

**Issue by ThomasPujolle** [LaserWeb/LaserWeb4-Binaries#35]
There is something I probably don't understand, but for each latest release there is often .exe and .yaml, but no .dmg, how can I manage to get those? Thanks

**Comment by wndxlori** [comment]
Me too.

**Comment by albworkshop** [comment]
same issue for me, cannot find the DMG file  to download

**Comment by smoki3** [comment]
Also looking for this

**Comment by wndxlori** [comment]
Ok, just to save other people the time... you can run it from your browser here:

https://laserweb.github.io/LaserWeb4/

That's the only solution I found. Not sure why that's not in the documentation. It would save a lot of time and frustration.

**Comment by 3Dietrich** [comment]
Either you don't support "OSX" or you need to answer: **Where is the .DMG file**?
The last file I found is "untagged-.." what does it mean. It is version v4.0.996-130 - from 2018. Is this the correct DMG file ?

**Comment by cprezzi** [comment]
@3Dietrich Yes, this is the last DMG that was compiled. The dev who was responsible for comiling the MAC DMG has left the project and nobody else has offered to take this part over yet.
